build_tools/github_actions/tests/configure_ci_test.py

```diff
@@ -184,23 +173,10 @@ def test_main_linux_branch_push_matrix_generator(self):
             families={},
             platform="linux",
         )
-        linux_target_output.sort(key=lambda item: item["family"])
-        linux_target_to_compare = [
-            {
-                "test-runs-on": "linux-rx9070-gpu-rocm",
-                "family": "gfx120X-all",
-                "pytorch-target": "gfx1201",
-            },
-            {
-                "test-runs-on": "linux-mi300-1gpu-ossci-rocm",
-                "family": "gfx94X-dcgpu",
-                "pytorch-target": "gfx942",
-            },
-            {"test-runs-on": "", "family": "gfx110X-dgpu", "pytorch-target": "gfx1100"},
-            {"test-runs-on": "", "family": "gfx1151", "pytorch-target": "gfx1151"},
-        ]
-        linux_target_to_compare.sort(key=lambda item: item["family"])
-        self.assertEqual(linux_target_output, linux_target_to_compare)
+        self.assertGreaterEqual(len(linux_target_output), 1)
+        self.assertGreater(len(linux_target_output), len(DEFAULT_LINUX_CONFIGURATIONS))
```

Comment:
I'm on the fence on this check. This asserts that `push` includes more targets than "default". If we had enough runner capacity, we could run the same set. Not sure how useful it is to test like this.